browser-extension/common/rollup.config.js

```diff
@@ -77,4 +77,22 @@ export default [
       }),
     ],
   },
+  {
+    input: "src/constants.ts",
```

Comment:
this is not used currently, but in future we would like MV2 to constants defined in this file to avoid duplication.